purchase_order/purchase_order.tpl

```diff
@@ -325,7 +327,9 @@ Expected: {{Order.arrivalDate|correcttimezone|date ("m/d/y")}}<br />
 {% if hide_general_notes == false %}
 {% if Order.Note.note|strlen > 0 %}
 <h3>Notes:</h3>
-<br>{{Order.Note.note}}
+<br>{% if note_line_break == true %}
+<pre id="note-section">{{Order.Note.note|raw}}</pre>{% else %}
```

Comment:
would this smarty modifyer do it? `|nl2br|raw`

Reply:
It does work! thats perfect 😄